channels/ru.m3u

```diff
@@ -1284,3 +1284,17 @@ http://live.ugratv.cdnvideo.ru/ugratv/smil:ugrastream1.smil/chunklist_b7192000.m
 http://yurgan.bonus-tv.ru/cdn/yurgan/tracks-v2a1/index.m3u8
 #EXTINF:-1 tvg-id="YuTV.ru" tvg-name="ЮТВ" tvg-country="RU" tvg-language="Russian" tvg-logo="" group-title="Local",ЮТВ (Чебоксары) (432p)
 http://serv24.vintera.tv:8081/utv/Stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",BRIDGE TV
+http://mfe.cliptv.az/dash/Bridge_TV_SD.ism/playlist.mpd
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",BRIDGE TV Dance
+http://mfe.cliptv.az/dash/Bridge_TV_Dance_SD.ism/playlist.mpd
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",XXI
+http://mfe.cliptv.az/dash/TV_XXI_SD.ism/playlist.mpd
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Mezzo Live
+http://mfe.cliptv.az/dash/Mezzo_Live_HD.ism/playlist.mpd
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV1000
+http://mfe.cliptv.az/dash/TV1000_SD.ism/playlist.mpd
```

Comment:
This is Swedish channel: https://www.lyngsat.com/tvchannels/se/TV-1000-East.html